Excel_README.md

```diff
@@ -1,57 +1,59 @@
-# TASBE Flow Analytics Excel Interface Instruction Guide
+# TASBEFlowAnalytics Excel Interface Instruction Guide
```

Comment:
Actually, the spaces are OK: that's what we call it in the releases.  We just can't have whitespace in the repository name.